masterlist.yaml

```
@@ -21262,13 +21262,27 @@ plugins:
         nav: 1
 
   - name: 'RedBag''s Solitude.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/42052/' ]
-    dirty:
-      - <<: *quickClean
-        crc: 0xC025F482
-        util: '[SSEEdit v4.0.3g](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
-        itm: 95
-        udr: 79
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/42052/'
+        name: 'Redbag''s Solitude'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64068/'
+        name: 'Rob''s Bug Fixes - RedBag''s Solitude'
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[Rob''s Bug Fixes - RedBag''s Solitude](https://www.nexusmods.com/skyrimspecialedition/mods/64068/)' ]
+        condition: 'checksum("RedBag''s Solitude.esp", A9A23AB8)'
+    clean:
+      - crc: 0xA9A23AB8
+        util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+      - crc: 0x90AD69F2
+        util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
```

The `util` string for clean cleaning data doesn't support Markdown (only dirty ones do), you can just copy & paste what xEdit reports after QAC:

```suggestion
      - crc: 0xA9A23AB8
        util: 'SSEEdit v4.0.4b'
      - crc: 0x90AD69F2
        util: 'SSEEdit v4.0.4b'
```